lapack.go

```diff
@@ -24,6 +24,7 @@ type Complex128 interface{}
 // Float64 defines the public float64 LAPACK API supported by gonum/lapack.
 type Float64 interface {
 	Dgecon(norm MatrixNorm, n int, a []float64, lda int, anorm float64, work []float64, iwork []int) float64
+	Dgehrd(n, ilo, ihi int, a []float64, lda int, tau, work []float64, lwork int)
```

Comment:
Does this need to be part of the interface? To date we've only exported those that are used in mat64 directly.